piuma-image-optimizer.php

```diff
@@ -17,3 +35,77 @@
 
 
 $piumaImageOptimizer = new piumaImageOptimizer();
+
+function piuma_image_optimizer_init()
+{
+?>
+
+
+    <div class="wrap">
+        <div id="update-success" class="notice notice-success is-dismissible">
+            <p>settings updated</p>
+        </div>
+
+        <div id="reset-success" class="notice notice-success is-dismissible">
+            <p>settings are resetted</p>
+        </div>
+
+
+        <div class="title-area">
+            <img itemprop="image" class="TableObject-item avatar flex-shrink-0" src="https://avatars3.githubusercontent.com/u/56169391?s=200&amp;v=4" width="100" height="100" alt="@piumaio">
+            <h1>Piuma Image Optimizer</h1>
+        </div>
+        <div class="panel">
+
+
+            <div class="initial_form box">
+
+
+                <div class="input-text-wrap">
+                    <label for="base_remote_url">Your Piuma url</label>
+                    <input type="url" pattern="https?://.*" name="base_remote_url" id="base_remote_url" value="<?= get_option('piuma_base_remote_url') ?>">
+                </div>
+                <hr>
+                <label for="img_resize_height">You image Height</label>
+                <input type="number" name="img_resize_height" id="img_resize_height" min="0" value="<?= get_option('piuma_img_resize_height') ?>">
+                <label for="img_resize_width">You image Width</label>
+                <input type="number" name="img_resize_width" id="img_resize_width" min="0" value="<?= get_option('piuma_img_resize_width') ?>">
```

Comment:
qui pensandoci bene forse non vanno impostate le dimensioni "fisse", ma le dimensioni massime, eventualmente, perchè nel pannello il plugin in questo va a definire le dimensioni di tutte le immagini del sito.
Se invece ci riferiamo a valori massimi, allora si può essere utile.

Reply:
io avevo capito che si intendessero come valori massimi, forse basta scrivercelo